api/embed.js

```diff
@@ -3,7 +3,7 @@
   var firstRun = !window.polis._hasRun;
   polis._hasRun = 1;
   var iframes = [];
-  var polisUrl = "https://pol.is";
+  var polisUrl = "https://ait-polis.pdis.nat.gov.tw";
```

Comment:
This should really just be a config variable if there isn't one already.

Reply:
Yeah, but I don't know how to do it in webpack.